homeassistant/components/sensor/fail2ban.py

```diff
@@ -77,19 +82,21 @@ def name(self):
     @property
     def state_attributes(self):
         """Return the state attributes of the fail2ban sensor."""
+        self.ban_dict[STATE_TOTAL_BANS] = self.total_bans
         return self.ban_dict
 
     @property
     def state(self):
         """Return the most recently banned IP Address."""
-        return self.last_ban
+        return len(self.ban_dict[STATE_CURRENT_BANS])
```

Comment:
We have this and there is `self.total_bans`, which one is correct?

Also, when making it a number, also add the `unit_of_measurement` property